TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsTableViewManager.swift

```diff
@@ -50,51 +61,72 @@ class TradeItPortfolioPositionsTableViewManager: NSObject, UITableViewDelegate,
     }
     
     private func reloadTableViewAtIndexPath(_ indexPaths: [IndexPath]) {
-        self.positionsTable?.beginUpdates()
-        self.positionsTable?.reloadRows(at: indexPaths, with: .automatic)
-        self.positionsTable?.endUpdates()
-        self.positionsTable?.selectRow(at: indexPaths.last, animated: true, scrollPosition: .top)
+        self.table?.beginUpdates()
+        self.table?.reloadRows(at: indexPaths, with: .automatic)
+        self.table?.endUpdates()
     }
 
     // MARK: UITableViewDataSource
-    
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return SECTIONS.count.rawValue
+    }
+
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return self.positions.count
+        if section == SECTIONS.accountDetails.rawValue {
+            return 1
+        } else {
+            return self.positions.count
+        }
     }
     
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        var cell: UITableViewCell?
+        if section == SECTIONS.accountDetails.rawValue {
+            return nil
+        } else {
+            let cell = tableView.dequeueReusableCell(withIdentifier: "PORTFOLIO_EQUITY_POSITIONS_HEADER_ID")
+            //TradeItThemeConfigurator.configureTableHeader(header: cell)
+            return cell?.contentView
+        }
+    }
 
-        if self.positions.count > 0 {
-            if self.positions[0].position != nil {
-                cell = tableView.dequeueReusableCell(withIdentifier: "PORTFOLIO_EQUITY_POSITIONS_HEADER_ID")
-            } else if self.positions[0].fxPosition != nil {
-                cell = tableView.dequeueReusableCell(withIdentifier: "PORTFOLIO_FX_POSITIONS_HEADER_ID")
-            }
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        if section == SECTIONS.accountDetails.rawValue {
+            return 0.0
+        } else {
+            return 36.0
         }
-        
-        TradeItThemeConfigurator.configureTableHeader(header: cell)
-        return cell
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let position = self.positions[indexPath.row]
-        let cell = self.provideCell(forTableView: tableView,
-                                    forPortfolioPosition: position,
-                                    selected: self.selectedPositionIndex == indexPath.row)
-        return cell
+        if indexPath.section == SECTIONS.accountDetails.rawValue {
+            if let account = self.account {
+                let cell = tableView.dequeueReusableCell(withIdentifier: "TRADE_IT_PORTFOLIO_ACCOUNT_DETAILS") as! TradeItPortfolioAccountDetailsTableViewCell
+                cell.populate(withAccount: account)
+                return cell
+            } else {
+                return UITableViewCell()
```

Comment:
Change this to `guard let account = self.account else { return UITableViewCell() }` or whatever...